### ISSUE TITLE
iOS 14 WKWebView Most Times Will Not Render Page Or Will Flicker

### ISSUE THREAD

**Issue by geeetarcode** [xamarin/Xamarin.Forms#12421]
### Description

Supporting two iOS apps. iOS App #1: WebViews will not render and show a blank page. Endpoints render fine in a browser on device. iOS App #2 Some endpoints will render and others will flicker (render and turn white). This is only happening on iOS 14.

### Steps to Reproduce

1. Navigate to webpage endpoint
2. See some web pages show up blank/white in WebView

### Expected Behavior

Web pages should render as normal like they do with iOS 13 and below.

### Actual Behavior

WebView render a blank/white page for some (on app #1) or all (for app #2) Url's.

### Basic Information

- Version with issue: iOS 14
- Last known good version: iOS 13 and below
- IDE: Visual Studio for Mac 8.7.8, Xcode 12
- Platform Target Frameworks: 
  - iOS:  14.0
- Nuget Packages:
- Affected Devices: All iOS devices on iOS 14

Wondering if this has anything to do with Xamarin dealing with WKWebView on iOS 14?

**Comment by jsuarezruiz** [comment]
@geeetarcode I want to do some tests, could you share some of the URLs you have problems with?

**Comment by geeetarcode** [comment]
@jsuarezruiz  Here are some example links: 

Thanks for following up! Here are some example Urls that aren't working correctly on iOS 14 only, fine on all other versions:

App #1:

This Url launches, flickers the page and goes white/blank:

https://www.musiciansfriend.com/new-taylor-grand-theater-guitars?hideNav=true&mobile=yes&icid=219328&adobe_mc=MCMID%3D15878712815986016550100092610096577045

This Url does work, same app: 

https://www.musiciansfriend.com/pdp/reviews/productReviewsDisplay4.jsp?hideNav=true&noHeaderFooter=true&productId=J19628&adobe_mc=MCMID%3D15878712815986016550100092610096577045

In app #1, only reviews work correctly...everything else flickers and goes blank/white.

App #2:

This Url doesn't work, goes blank/white:

http://www.guitarcenter.com/Services/Rentals/?hideNav=true&mobile=yes&icid=407716&adobe_mc=MCMID%3D80659864000117219880552394521748643832


This review page that works on app #1 does not on app #2, like the above, it just goes blank/white:


https://www.guitarcenter.com/pdp/reviews/productReviewsDisplay4.jsp?hideNav=true&noHeaderFooter=true&productId=L56573&adobe_mc=MCMID%3D80659864000117219880552394521748643832


We verified that this issue is happening on all versions of our app on iOS 14--this isn't due to recent changes or anything like that. 


I'm wondering if it may have to do with Intellegent Tracking Prevention, which is new for iOS 14. This is my best guess as to why it would only be affecting iOS 14 and fits with the timeline of when the issue started presenting. We've not been able to verify yet. 

https://www.simoahava.com/privacy/intelligent-tracking-prevention-ios-14-ipados-14-safari-14/

**Comment by geeetarcode** [comment]
I ended up finding the issue. It wasn't a Xamarin issue, per se. What happened was that there was something with iOS 14 that broke our custom WebViewRenderer. I traced it and found the fix. But, it did puzzle me as to why it would happened except breaking changes that affected the Xamarin codebase. If someone else runs into something similar, it might save time going straight to the WebViewRenderer if it's been done custom.
